xapian-core/tests/api_backend.cc

```diff
@@ -169,7 +169,8 @@ DEFINE_TESTCASE(dbstats1, backend) {
 	TEST_REL(db.get_doclength_lower_bound(),<=,min_len);
     }
 
-    if (get_dbtype() != "inmemory" && !startswith(get_dbtype(), "remote")) {
+    if (get_dbtype() != "inmemory" && !startswith(get_dbtype(), "remote")
+        && !endswith(get_dbtype(), "remoteprog_glass")) {
```

Comment:
Xapian coding style is to wrap an expression after the operator, so the `&&` should be on the end of the first line here, not the start of the second.

But I think probably the test here should be that "remote" is not a substring of the dbtype (so `get_dbtype().find("remote") == string::npos` or something like that).  That should get all the cases it should without any false matches in practice, and avoids a growing list of more precise checks.